test/common_quantized.py

```diff
@@ -65,8 +65,12 @@ def _calculate_dynamic_qparams(X, dtype):
 
 @contextmanager
 def enable_mobile_quantized_engine():
-    torch.backends.quantized.engine = torch.qnnpack
+    torch.backends.quantized.engine = "qnnpack"
     try:
         yield
     finally:
-        torch.backends.quantized.engine = torch.fbgemm
+        qengines = torch._C._supported_qengines()
+        if 2 in qengines:
```

Comment:
oh, you will need to change this as well

Reply:
oops, missed this.